### PR TITLE
Initial 

### PR DESCRIPTION
**Pull request by kalpgarg**

### CODE REVIEW

**Comment by agshubhs** [comment]
@kalpgarg It's looking nice.
Before merging it need a change.
1. Add names of team member and mentor name in Readme file in alphabetical order.
2.  Electronics part and arduino is not present in it.
3. What does this src folder signify?

**Comment by palnabarun** [comment]
@Shubhanshu07 `src` is generally used to store source codes of a project. Ideally, every project should have used this folder to store the code.

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Abstract
+Our idea is to implement a robot which can perform SLAM (Simultaneous Localization and Mapping) autonomously in any surrounding. This automated bot can be deployed in various situations which are risky for humans such as in remote survelliance, etc. 
+
+# Pre-requisite and Software Used
+This whole work is being developed in ROS(Robot Operating System). It is being chosen due to its simple usage to implement various different components and pacjkages used for robot  building and to integrate them easily. To get basic ROS usage guidelines, refer this link: [ROS Documentaion](http://wiki.ros.org/) </br>
```
Comment (palnabarun) [inline readme.md]: Please remove the <br/> tags. Paragraphs in markdown are denoted by a black line.

**Review by palnabarun — CHANGES_REQUESTED** [review]

**Comment by palnabarun** [comment]
@kalpgarg Please try to put some effort into the commit messages.

**Review by agshubhs — APPROVED** [review]

**Comment by agshubhs** [comment]
It looks good to me.

**Pull request merged**